strawberryfields/apps/plot.py

```diff
@@ -102,34 +97,36 @@ def _edge_coords(g: nx.Graph, l: dict) -> dict:
 
 def graph(g: nx.Graph, s: Optional[list] = None, plot_size: Tuple = (500, 500)):  # pragma: no cover
```

Comment:
These empty lines shouldn't be removed. Some of them (e.g., the ones the `.. image::` directive below) are causing the docs build to fail. Not sure if these were somehow removed when you attempted to format the file, but they need to be added back before merging.